source/wp-content/themes/wporg-showcase-2022/functions.php

```diff
@@ -39,6 +40,8 @@
 add_filter( 'grunion_should_send_email', '__return_false' );
 // Enable auto-fill using user information.
 add_filter( 'jetpack_auto_fill_logged_in_user', '__return_true' );
+// Remove Jetpack CSS on frontend
+add_filter( 'jetpack_implode_frontend_css', '__return_false', 99 );
```

Comment:
See https://css-tricks.com/snippets/wordpress/removing-jetpack-css/

Reply:
Good find. Looping in @pkevan since he has more experience with JetPack than I do. The front-end and form still appear to work as expected. 👍

Reply:
Yeah the [Jetpack docs](https://developer.jetpack.com/hooks/jetpack_implode_frontend_css/) describe the intention of this quiet differently, but I've found multiple sources using it for this purpose.

Reply:
When there is something like this found, it's usually best to highlight it with Jetpack themselves, as it could be an opportunity on their side to optimise or a bug in the implementation itself.

Reply:
> highlight it with Jetpack themselves

Good call, I'll raise this and see if there's a better way to remove this CSS too, as neither `dequeue` or `deregister` work.